ui/app/components/clients/page/token.hbs

```diff
@@ -34,7 +33,6 @@
   <Clients::ChartContainer
     @title="Monthly new"
     @description="Entity or non-entity clients which interacted with Vault for the first time during this date range. Each bar represents the total new clients for that month."
-    @timestamp={{@activity.responseTimestamp}}
```

Comment:
Now that these charts are the same, one should be deleted. It looks like a combination of the each chart descriptions would be the most accurate